pkg/util/ranger/detacher.go

```diff
@@ -487,7 +487,9 @@ func (d *rangeDetacher) detachCNFCondAndBuildRangeForIndex(conditions []expressi
 // excludeToIncludeForIntPoint converts `(i` to `[i+1` and `i)` to `i-1]` if `i` is integer.
 // For example, if p is `(3`, i.e., point { value: int(3), excl: true, start: true }, it is equal to `[4`, i.e., point { value: int(4), excl: false, start: true }.
 // Similarly, if p is `8)`, i.e., point { value: int(8), excl: true, start: false}, it is equal to `7]`, i.e., point { value: int(7), excl: false, start: false }.
-// If return value is nil, it means p is unsatisfiable. For example, `(MaxInt64` is unsatisfiable.
+// If return value is nil, it means p is unsatisfiable. For example, `(MaxUint64` is unsatisfiable.
+// The boundary value will be treated as the bigger type: For example, `(MaxInt64` of type KindInt64 will become `[MaxInt64+1` of type KindUint64,
```

Comment:
If so, the comment should also be updated. Or we pass the column's field type to this function, to do a more accurate judgment.

Reply:
@onlyacat Could you check this review？

Reply:
> If so, the comment should also be updated. Or we pass the column's field type to this function, to do a more accurate judgment.

> @onlyacat Could you check this review？

Sorry for the late reply. 
Since this is just an early termination step, I guess it is not a bad idea to keep it simple and the unsatisfied value for specific field type will be eliminated in the later stage.
As for the comment, it described what I edited as actual. Not sure which should be updated.